REAC-6-lista-hotela

AhmedMehmedovic opened the pull request:
Izrada prikaza lista hotela uz odabir u search polju opcije hotels, izbor typa pretrage, ucitavanje hotela iz local storage te rendanje istih u listi gdje se prilikom pretrage prikazuju i rezultati pretrage. Ispravka u rulesValidator.js regex validacije, i uslova.
Brisanje iz importa nepotrebnih filova  hotel.js i listHotelView.js

AhmedMehmedovic committed:
REAC-6-lista-hotela
Izrada prikaza lista hotela uz odabir u search polju opcije hotels, izbor typa pretrage, ucitavanje hotela iz local storage te rendanje istih u listi gdje se prilikom pretrage prikazuju i rezultati pretrage. Ispravka u rulesValidator.js regex validacije, i uslova.

AhmedMehmedovic committed:
REAC-6-lista-hotela
Brisanje iz importa nepotrebnih filova  hotel.js i listHotelView.js

AhmedMehmedovic committed:
REAC-6
Prepravka _preview scss u prewiew data komentarisan display: grid;

AhmedMehmedovic committed:
REAC-6
Prepravka u model na liniji 76 prilikom pretrage, pretvaram qurey u lowercase i iz baze ono sto pretrazujem te tako mogu prikazati rezultate bez obzira da li u pretrazi unosili mala slova ili velika

maniaba reviewed (changes requested):
Poprabno je da prilkom pretrage hotela kada nema rezulltata izmjenis poruku
(No recepies found for youre query! Please try again!)

maniaba reviewed (changes requested):
greške validacije u modulau formatiraj (LJepši prikaz)

maniaba commented:
Popraviti unos novih hotela

maniaba commented:
Auto zavaranje modala na pretrazi

AhmedMehmedovic committed:
REAC-6
Ispravke nakon pregleda. Ispravka validatora, ispravka poruke da nije pronadjen (hotel/recept), ispravka hover na dropdown meni.
